src/views/confirm-email.njk

```diff
@@ -0,0 +1,102 @@
+{% extends "_base.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set emailAddressError %}
+{% if model.missingConfirmEmailAddressValue %}
+      Enter an email address
+    {% elif model.invalidEmailAddressValue %}
+      Enter an email address in the correct format, like name@example.com
+    {% else %}
+	{# Blank. Not an error. #}
+{% endif %}
+{% endset %}
+
+{% block content %}
+
+	{% if model.emailConfirmError %}
+		{{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          text: "Confirm if the stated email is correct or not",
```

Comment:
I think, from the issue, this error is meant to be `Check and confirm your email address`